openlibrary/components/LibraryExplorer/components/LibraryToolbar.vue

```diff
@@ -2,41 +2,41 @@
     <div class="floating-controls-wrapper">
       <div class="floating-controls" :class="{open: openTabs.length > 0}">
         <div class="tab-bar">
-          <div class="chunky-icon" v-if="openTabs.length" @click="openTabs.splice(0, openTabs.length)">
+          <button v-if="openTabs.length" @click="openTabs.splice(0, openTabs.length)">
```

Comment:
We'll still need the class `chunky-icon` here; otherwise a lot of the CSS won't apply!

Reply:
Actually, let's make it `chunky-icon tab-bar--tab`; that'll make the CSS easier to understand.

Reply:
@cdrini What is the added value of adding class `tab-bar--tab`? To mark it as a child of `tab-bar`?
I'm no CSS expert but my initial idea was to simplify the style sheet as `chunky-icon` class was not used anymore (Less is more, or maybe not...).
Anyway, I found some doc on BEM methodology. I'll dig more into that for my knowledge.

Reply:
The main benefit for me is CSS readability; this is more clearly each "tab" component of the tab bar, whereas `.tab-bar > button` isn't as clear if you're just looking at the CSS.